core/wasmWasi/src/internal/TimeZonesInitializer.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019-2023 JetBrains s.r.o. and contributors.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime.internal
+
+@RequiresOptIn
+internal annotation class InternalDateTimeApi
+
+/*
+This is internal API which is not intended to use on user-side.
+ */
```

Comment:
I think Dokka will generate the HTML for this API entry if we don't `/** @suppress */` it, no?

Reply:
Yes I think so. Do you know the way how to escape it?

Reply:
It will not, because all declarations matching `.*\.internal(\..*)?` regex are suppressed.

Reply:
But now dokka NOT generates `wasm` targets (both) and generates intermediate targets `commonJs` and `pureKotlin`

Reply:
Yeah, you are right. It's strange that it generates something for `pureKotlin` and `commonJs`, but this PR has nothing to do with it. Thanks!